src/server/input/default_input_manager.cpp

```diff
@@ -16,6 +16,8 @@
  * Authored by: Andreas Pokorny <andreas.pokorny@canonical.com>
  */
 
+#include <memory>
+
 #include "default_input_manager.h"
```

Comment:
We have a policy of including the "corresponding" header first (in this case `#include "default_input_manager.h"` should be the first include).

Please put `#include <memory>` after `#include <future>`